### DISCUSSION TITLE
Need etcdctl CLI tool to work with Kubernetes etcd

### DISCUSSION THREAD

**Discussion by eliassal** [etcd-io/etcd#15434] [General]
HI, I have kubernetes cluster up and running and understood that to work with etcd I need etcdctl CLI lets say for backup purposes. I am not able to see where I can download this cli tool.
Thanks for your help

**Comment by jmhbnz** [comment]
Hey @eliassal - The `etcdctl` cli is bundled with the `etcd` release on github, you can download the latest release, for example 3.5.7 here: https://github.com/etcd-io/etcd/releases/tag/v3.5.7. 

Once you follow the instructions in the release page to download and extract this you will have an `etcdctl` cli binary available to use.

Refer to docs here which cover this also: https://etcd.io/docs/v3.5/install/#install-pre-built-binaries

**Reply by eliassal** [reply]
Thanks
